virttest/shared/cfg/guest-os/Linux/CentOS-Stream/9.cfg

```diff
@@ -0,0 +1,44 @@
+- 9:
```

Comment:
For CentOS-Stream9, I'm getting an installation error: "Not enough space in file systems for the current software selection. An additional 1.71GiB is needed".

On the Avocado-VT side, it means having a larger `image_size` for this distro and corresponding KS file.  This works on my system:

```
avocado run --vt-guest-os Guest.Linux.CentOS-Stream.9.devel.x86_64 --vt-extra-params 'url=http://mirror.stream.centos.org/9-stream/BaseOS/x86_64/os/' --vt-extra-params 'image_size=15G' -- io-github-autotest-qemu.unattended_install.url.extra_cdrom_ks.default_install.aio_native
```